mmv1/third_party/terraform/website/docs/r/composer_user_workloads_secret.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+subcategory: "Cloud Composer"
+description: |-
+  Kubernetes secret in Composer Environment workloads.
```

Comment:
"in Composer Environment workloads" sounds for me a bit weird, because secret is actually not workload.

I was thinking that we could put here the same as this resource is described in public Composer API:
"User workloads Secret used by Airflow tasks that run with Kubernetes executor or KubernetesPodOperator."

Please, note that the decision was to use capital "Secret" and "ConfigMap" everywhere in our public documentation to be consistent how it is referenced in kubernetes documentation https://kubernetes.io/docs/concepts/configuration/secret/

Reply:
Addressed in separate [PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/10490) since this one is closed.